cmd/monaco/download/download_configs.go

```diff
@@ -202,14 +224,14 @@ func doDownloadConfigs(fs afero.Fs, downloaders downloaders, opts downloadConfig
 func downloadConfigs(downloaders downloaders, opts downloadConfigsOptions) (project.ConfigsPerType, error) {
 	configs := make(project.ConfigsPerType)
 
-	if shouldDownloadClassicConfigs(opts) {
+	{
 		log.Info("Downloading configuration APIs")
 
-		classicAPIs := makeClassicAPIs(opts.specificAPIs)
-		classicCfgs, err := downloaders.Classic().Download(opts.projectName, classicAPIs...)
+		classicCfgs, err := downloaders.Classic().Download(opts.projectName)
 		if err != nil {
 			return nil, err
 		}
+		log.Info("downloaded %q configurations from classic Config API endpoints", len(classicCfgs))
```

Comment:
nit: don't think that number needs to be in quotes
```suggestion
		log.Info("downloaded %s configurations from classic Config API endpoints", len(classicCfgs))
```